src/grafana/deploy/grafana-configuration/pai-jobview-dashboard.json.template

```diff
@@ -227,14 +227,14 @@
               "label": "GB",
               "logBase": 1,
               "max": null,
-              "min": null,
+              "min": 0,
```

Comment:
What's the difference for 0 and null?

Reply:
The yaxis may begin from negative values if set as null.